src/MessageHandler/Analytics/AnalyticsMessageHandler.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace App\MessageHandler\Analytics;
+
+use App\Message\Analytics\AnalyticsMessageBase;
+use App\MessageTransformer\Analytics\GURaaSMessageTransformer;
+use Symfony\Component\Messenger\Attribute\AsMessageHandler;
+use Symfony\Component\Uid\Uuid;
+use Symfony\Contracts\HttpClient\Exception\TransportExceptionInterface;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+#[AsMessageHandler]
+class AnalyticsMessageHandler
+{
+    private Uuid $guraasGameId;
+    private GURaaSMessageTransformer $guraasRequestTransformer;
+    private HttpClientInterface $httpClient;
+
+    public function __construct(
+        HttpClientInterface $httpClient
+    ) {
+        $this->guraasGameId = Uuid::fromString('3318cf30-f78d-4284-b530-a329005c248a');
```

Comment:
Add Game Id as parameter in the services.yaml https://symfony.com/doc/current/configuration.html#configuration-parameters